### ISSUE TITLE
Installing bitcoind from ppa may require universe repo

### ISSUE THREAD

**Issue by MrHash** [ElementsProject/elementsproject.github.io#29]
Following `bitcoind` installation instructions here: https://elementsproject.org/elements-code-tutorial/installing-bitcoin

It was necessary to run  `sudo add-apt-repository universe` with Ubuntu 18.04.1 Server in order for `bitcoind` dependencies to be located.

**Comment by wintercooled** [comment]
So can I just check @MrHash - you couldn't run the following without error:

```
sudo add-apt-repository ppa:bitcoin/bitcoin
```

Until you had first added the universe repository by running:

```
sudo add-apt-repository universe
```

I can add a note to the page to mention this extra step for those running the Server edition if you can confirm the above. Thanks, Matt.

**Comment by MrHash** [comment]
Not quite, i could not run `sudo apt-get install bitcoind` since `libzmq5` could not be found so i'd recommend a note to add the universe repo before `sudo apt-get update`.

**Comment by wintercooled** [comment]
Merged PR #31 to fix.
Tested locally on VM with Ubuntu 18.04.01 clean install.
Thanks @MrHash
